src-css/_bar.scss

```diff
@@ -175,20 +177,25 @@
 	}
 
 	// Displaying mv-bar in small spaces
-	&.mv-compact {
-		.mv-status > span {
-			display: none;
+	@container mv-bar (width <= 40em) {
```

Comment:
`40em` and `25em` are magic numbers obtained from experiments that “feel OK” in general cases. However, it's not the silver bullet for all the cases. If the Mavo bar has many buttons, these numbers need to be adjusted.

Reply:
One thing at a time. It's ok to keep the magic numbers for now. Eventually we want to move to a component that handles its own layout.